package.json

```diff
@@ -39,7 +39,8 @@
     "test.karma": "karma start",
     "prepare": "husky install",
     "prerelease": "patch-package",
-    "release": "yarn prerelease && yarn workspaces foreach run release"
+    "release": "yarn prerelease && yarn workspaces foreach run release",
+    "jest": "jest"
```

Comment:
Added the "jest" script so that it will be easier to run jest from the command line such that the version installed in node_modules will be used instead of whatever version of jest installed on a particular machine.